docs/identity/role-based-access-control/permissions-reference.md

```diff
@@ -1416,7 +1416,7 @@ Users with this role **cannot** do the following:
 > | microsoft.directory/roleAssignments/allProperties/read | Read all properties of role assignments |
 > | microsoft.directory/roleDefinitions/allProperties/read | Read all properties of role definitions |
 > | microsoft.directory/scopedRoleMemberships/allProperties/read | View members in administrative units |
-> | microsoft.directory/serviceAction/getAvailableExtentionProperties | Can perform the getAvailableExtentionProperties service action |
```

Comment:
Please revert this change

Reply:
Reverted. Is this something you can bring back up the tree so they fix it ?
As the source is spelled correctly - https://learn.microsoft.com/en-us/graph/api/directoryobject-getavailableextensionproperties?view=graph-rest-1.0&tabs=http